db/static_seeds.rb

```diff
@@ -8,3 +8,6 @@
   {start_date: "2017-05-28", name: "Bio", video: nil, duration: nil},
   {start_date: "2017-05-28", name: "Konventionell", video: nil, duration: nil}
 ])
+Sensor.create!([
```

Comment:
You create sensors in tests just as usual, ie. `create(:sensor, smaxtecs_sensor: true)`, for production, the best option would be to have a user interface where you can create sensors (and provide the `animal_id`)